docs/source-pytorch/common/checkpointing_basic.rst

```diff
@@ -197,16 +204,31 @@ You can disable checkpointing by passing:
 
 ----
 
+
 *********************
 Resume training state
 *********************
 
 If you don't just want to load weights, but instead restore the full training, do the following:
 
+.. warning::
```

Comment:
Can we move the warning after the correct usage? Let's first describe the right way to go about it, then show the warning.